tendrl/alerting/handlers/__init__.py

```diff
@@ -0,0 +1,123 @@
+import datetime
+import etcd
+import importlib
+import inspect
+import logging
+import os
+import six
+from tendrl.alerting.objects.alert import AlertUtils
+from tendrl.commons.singleton import to_singleton
+
+LOG = logging.getLogger(__name__)
+
+
+class NoHandlerException(Exception):
+    pass
+
+
+class HandlerMount(type):
+
+    def __init__(cls, name, bases, attrs):
+        if not hasattr(cls, 'handlers'):
+            cls.handlers = []
+        else:
+            cls.register_handler(cls)
+
+    def register_handler(cls, handler):
+        instance = handler()
+        cls.handlers.append(instance)
+
+
+@six.add_metaclass(HandlerMount)
+class AlertHandler(object):
+    def __init__(self):
+        self.time_stamp = datetime.datetime.now().isoformat()
+        self.alert = None
+        self.handles = ''
+
+    def update_alert(self, etcd_client):
```

Comment:
@shtripat You are right this was just WIP I pushed so that I don't loose the changes